src/modules/simulation/simulator_mavlink/SimulatorMavlink.cpp

```diff
@@ -464,6 +464,25 @@ void SimulatorMavlink::handle_message_hil_gps(const mavlink_message_t *msg)
 				break;
 			}
 		}
+
+
+#if 1
+		{
+			// hack for testing
+			vehicle_global_position_s aux_global_position{};
```

Comment:
This should come out before merging.
Do we want to keep a test like this somewhere in case it's needed?

Reply:
Hmm, yes we can't keep it like that but it would still be useful to have something for SITL testing.